comps/embeddings/langchain/requirements.txt

```diff
@@ -1,7 +1,8 @@
 docarray[full]
 fastapi
 huggingface_hub
-langchain
+langchain==0.2.12
```

Comment:
why fix the version?

Reply:
I wanted to show test are passing with the latest langchain but we can remove the version constraint

Reply:
let's remove the version limitation, we will freeze it in the release tag in our future release. And in the main branch, let's continue use the latest one.

Reply:
ok, removed it